README.md

```diff
@@ -1 +1,26 @@
-# node-repl-lab
\ No newline at end of file
+# Running JavaScript Code in Node REPL
+
+## Objectives
+
+1. Use REPL
```

Comment:
@vicfriedman it's better to use 1. 1. 1. in Markdown because the parser will automatically convert it to 1, 2, 3 if I remove or add a new number, i.e., we want to let the parser to the automatic numbering, not do it our selves

GitHub and any other parser converts 1, 1, 1, into 1, 2, 3 automatically, see the attached image
![screenshot 2016-02-08 08 29 37](https://cloud.githubusercontent.com/assets/1137314/12891855/4b33ea7a-ce3e-11e5-8fe8-69073ab2404c.png)